src/rules/unnecessaryConstructorRule.ts

```diff
@@ -15,18 +15,36 @@
  * limitations under the License.
  */
 
-import { isConstructorDeclaration, isParameterProperty } from "tsutils";
+import {
+    isCallExpression,
+    isConstructorDeclaration,
+    isExpressionStatement,
+    isParameterProperty,
+} from "tsutils";
 import * as ts from "typescript";
 
 import * as Lint from "../index";
 import { Replacement } from "../language/rule/rule";
 
+interface Options {
+    unnecessarySuperCall: boolean;
+}
+
+const OPTION_UNNECESSARY_SUPER_CALL = "unnecessary-super-call";
```

Comment:
Yeah, I'm not sold on this option name either... "Unnecessary super call" implies the `super` is the unnecessary thing, not the whole `constructor`. How about... `check-super-calls`?

Reply:
Done